tools/system_libs.py

```diff
@@ -872,7 +872,12 @@ def fetch_project(name, url, subdir, is_tarbz2=False):
           Ports.clear_project_build(name)
           return
 
-    fullpath = fullname + ('.tar.bz2' if is_tarbz2 else '.zip')
+    if is_tarbz2:
+        fullpath = fullname + '.tar.bz2'
```

Comment:
as mentioned in the other PR, indentation should be 2 spaces on these lines

Reply:
OK, fixed the indentation.

Also to help prevent these kinds of things in the future, I submitted #8359.